src/components/Restoration/RestorationTypeSummary.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <v-card flat id="restoration-type-summary" class="ma-4">
+
+      <div v-if="!isRestorationTypeValid" class="error-message pb-4">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text mx-1">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.RESTORATION_BUSINESS_NAME}` }"
+          >Return to this step to finish it</router-link>
+        </span>
+      </div>
+
+      <v-row>
+        <v-col class="px-4">
```

Comment:
How wide is this column? Can you show screenshots on a wide screen and narrow screen?

Reply:
Wide:
![wide](https://user-images.githubusercontent.com/122301442/217629303-7963d737-2385-41a9-be4f-0bad7e5b7166.PNG)

Narrow:
![narrow](https://user-images.githubusercontent.com/122301442/217629335-96218bf4-2d65-4169-af64-b164e0cafd73.PNG)

Reply:
Can you show a little more on the narrow screen, please? I want to see how the other sections display.

Reply:
![localhost_8080_businesses_create_restoration-review-confirm_id=BC0871154 accountid=3040 (1)](https://user-images.githubusercontent.com/122301442/217661841-b5c67bcb-436c-424f-b10b-53b3ba4801b1.png)

Reply:
Even narrower!

What I want to check is that keys and values stack correctly on mobile, ie:
![image](https://user-images.githubusercontent.com/10002889/217663324-da744f88-e896-450d-b503-128ab08d9bb7.png)

Reply:
![localhost_8080_businesses_create_restoration-review-confirm_id=BC0871154 accountid=3040 (2)](https://user-images.githubusercontent.com/122301442/217663867-c3a02c2d-864d-452e-b77a-c4b0f4f124c2.png)

Reply:
OK, nice!